kie-plugins-testing/src/test/java/org/drools/compiler/xpath/tobeinstrumented/model/PojoWithCollections.java

```diff
@@ -0,0 +1,40 @@
+package org.drools.compiler.xpath.tobeinstrumented.model;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+public class PojoWithCollections {
+
+    private final Collection fieldCollection;
+    private final List fieldList;
+    private final Set fieldSet;
+
+    public PojoWithCollections(Collection fieldCollection,
+                               List fieldList,
+                               Set fieldSet) {
+        super();
```

Comment:
redundant call to `super();` as it's automatically inserted by the compiler. Also this class is child of `Object`.